// error count: 1

export function test(err: any): any {
  const c = [1,2]
  return err.nothing
}
